backend/fszipper.py

```diff
@@ -0,0 +1,109 @@
+from backend.settings import DOWNLOAD_TEMP
```

Comment:
`DOWNLOAD_TEMP` need to be added in `settings.example.py`.

Reply:
And maybe rename it to `DOWNLOAD_FOLDER_PATH` for consistency with other settings.